pom.xml

```diff
@@ -1081,6 +1081,15 @@ under the License.
 				</executions>
 			</plugin>
 
+			<!-- Pull bundled transitive dependencies (e.g. Mini-KDC see DIRSHARED-134) -->
```

Comment:
Could we also keep the link for convenience? https://issues.apache.org/jira/browse/DIRSHARED-134